### DISCUSSION TITLE
Comment out a line in DNS settings -> Access settings

### DISCUSSION THREAD

**Discussion by RomchikL** [AdguardTeam/AdGuardHome#3829] [Q&A] [answered]
Is it possible to comment out a line in "Access settings" section?
e.g., I would like to split android phone IPs (1.1.1.1/24, 2.2.2.2/24, 3.3.3.3/24, ... --- Telecom operator CIDR) and router IPs (3.3.3.3/24, 4.4.4.4/24, ... --- Internet service provider CIDR).
something like:
#My phone
1.1.1.1/24
2.2.2.2/24
3.3.3.3/24
#My router
3.3.3.3/24
4.4.4.4/24

**Comment by ainar-g** [comment, marked as answer]
It is currently not possible to do this.  You can open a new issue, and we'll consider adding this feature.
